pages/index.tsx

```diff
@@ -37,15 +37,21 @@ const PARALLAXCONF = {
 };
 
 const Home: NextPage = () => {
-  const [loading, setLoading] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
   const scrollRef = useRef();
   const theme = useTheme();
 
+  /**
+   * Loads the Order page
+   */
   const redirectToOrder = () => {
-    setLoading(true);
+    setIsLoading(true);
     window.location.replace("/order");
   };
 
+  /**
+   * Scroll to Ref attribute
```

Comment:
Scrolls to bottom of the chat history for the most recent messages

Reply:
Both the homepage and order page have a scrollOnClick() function. Here, I'll change the description to 'Scrolls to Learn More' section of the homepage